pkl-commons-cli/src/main/kotlin/org/pkl/commons/cli/commands/BaseOptions.kt

```diff
@@ -141,6 +143,17 @@ class BaseOptions : OptionGroup() {
       )
       .associateProps()
 
+  val color: Color by
+    option(
+        names = arrayOf("--color"),
+        metavar = "<when>",
+        help =
+          "Whether to format messages in ANSI color. Possible values of <when> are \"never\", \"auto\", and \"always\"."
```

Comment:
style: single quotes wouldn't need escaping

Reply:
Fair shout. Force of habit from referring to Pkl enum values (where the `"`s are part of the actual value).